tests/scripts/thread-cert/config.py

```diff
@@ -373,7 +373,7 @@ def create_default_based_on_src_dst_ports_udp_payload_factory(master_key):
         src_dst_port_based_payload_factories={
             19788: mle_message_factory,
             61631: coap_message_factory,
-            1000: dtls_message_factory,
+            5684: dtls_message_factory,
```

Comment:
both `1000` and `5684` can have dtls traffic.

Reply:
Then shall we direct both ports to the same factory?